From 40188009bf14b1756603cbe3adafa897458976b6 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <sanyog.r.kale@intel.com>
Date: Thu, 2 Mar 2017 13:24:46 +0530
Subject: [PATCH 4128/4301] ASoC: Intel: Initialize SRAM base address for CNL
 platform

commit b78e9726a3c8165031c9d8bfa0397425936f1b8e from
git://git@github.com:01org/linux-intel-4.9.git

Initializing SRAM base address and size which is used by
debugfs property "fw_soft_regs_rd" to read FW soft registers from
SRAM.

Change-Id: I917ac18313b613ed879b1594eb5ed1806f05da21
Signed-off-by: Sanyog Kale <sanyog.r.kale@intel.com>
Reviewed-on: https://git-gar-1.devtools.intel.com/gerrit/14052
Reviewed-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
Tested-by: Avati, Santosh Kumar <santosh.kumar.avati@intel.com>
---
 sound/soc/intel/skylake/cnl-sst.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/sound/soc/intel/skylake/cnl-sst.c b/sound/soc/intel/skylake/cnl-sst.c
index 5910536..d9c7018 100644
--- a/sound/soc/intel/skylake/cnl-sst.c
+++ b/sound/soc/intel/skylake/cnl-sst.c
@@ -762,6 +762,10 @@ int cnl_sst_dsp_init(struct device *dev, void __iomem *mmio_base, int irq,
 	sst->fw_ops = cnl_fw_ops;
 	sst->addr.lpe = mmio_base;
 	sst->addr.shim = mmio_base;
+	sst->addr.sram0_base = CNL_ADSP_SRAM0_BASE;
+	sst->addr.sram1_base = CNL_ADSP_SRAM1_BASE;
+	sst->addr.w0_stat_sz = CNL_ADSP_W0_STAT_SZ;
+	sst->addr.w0_up_sz = CNL_ADSP_W0_UP_SZ;
 
 	sst_dsp_mailbox_init(sst, (CNL_ADSP_SRAM0_BASE + CNL_ADSP_W0_STAT_SZ),
 			CNL_ADSP_W0_UP_SZ, CNL_ADSP_SRAM1_BASE, CNL_ADSP_W1_SZ);
-- 
1.7.5.4

